dapp/app/ts/library/ethereum-browser-dependencies.ts

```diff
@@ -7,7 +7,7 @@ import { ReadonlyFetchJsonRpcDependencies } from './readonly-fetch-dependencies'
 declare global {
 	interface Window {
 		ethereum: {
-			sendAsync: (options: { jsonrpc: '2.0', method: JsonRpcMethod | 'eth_requestAccounts', params: readonly unknown[] }, callback: (error: Error, response: unknown) => void) => void
```

Comment:
why isn't eth_requestAccounts just a part of the union?

Reply:
It is non-standard.  It is a special addition that some web3 apps/extensions add (like MetaMask).  It isn't something that Geth/Parity/Nethermind respond to.